src/main/java/com/github/sardine/impl/SardineImpl.java

```diff
@@ -695,12 +695,18 @@ public DavQuota getQuota(String url) throws IOException
 		{
 			return null;
 		}
-		else
+		DavResource resource;
```

Comment:
I do not understand this change as the variable `resource` has no usages.

Reply:
The redudant `else` was removed (there is an early return in the `if` above). The  try-catch was added as the catched `URISyntaxException` was unhandled. The question is, if it should be handled differently than simply returning `null`.